src/zookeeperLock.ts

```diff
@@ -759,24 +761,27 @@ export class ZookeeperLock extends EventEmitter {
 
                             const sequence = this.filterLocks(locks)
                                 .map((l) => {
-                                    return ZookeeperLock.getSequenceNumber(l);
+                                    return {
+                                        key: l,
+                                        sequence: ZookeeperLock.getSequenceNumber(l)
```

Comment:
can we rename this prop to `sequenceNumber` perhaps or like `lockSequenceIndex`? doing `sequence[i].sequence` downstream seems a bit weird?